pkg/server/storage/mysql.go

```diff
@@ -55,6 +55,40 @@ func (my *MysqlStorage) QueryHostkeys() (ResultIterator, error) {
 	return &SqlResultIterator{Rows: rows}, nil
 }
 
+func (my *MysqlStorage) RecordGitHubMapping(mapping map[string]string) error {
+	// Prepare for batch insert
+	entries := make([]string, 0, len(mapping))
+	values := make([]interface{}, 0, len(mapping)*2)
+	for ssoIdentity, githubUser := range mapping {
+		// Create one set of values for each mapping
+		entries = append(entries, "(?, ?)")
+		// Append matching values for mapping
+		values = append(values, ssoIdentity)
+		values = append(values, githubUser)
+	}
+	// Create prepared statement with necessary number of (?, ?) values
```

Comment:
this comment is misleading, because you're not preparing a statement here, you're just constructing a query string. More on Go pkg statements: https://golang.org/pkg/database/sql/#Stmt